.idea/misc.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I'm curious about to XML files here. What are they doing, and why are they here? I don't have any experience with XML, so it would be cool if you shared what you're up to.